safe/gis/vector/prepare_vector_layer.py

```diff
@@ -358,6 +358,16 @@ def _remove_features(layer):
                 # i += 1
                 pass
 
+            # If there is exposure that mapped to do not report class
+            if layer_purpose == layer_purpose_exposure['key']:
+                is_do_not_report = layer.keywords.get('value_map', {}).get(
+                    'do_not_report')
```

Comment:
And use this constant here ? `DO_NOT_REPORT` ?
Like this, all our strings are in definitions